### PR TITLE
[Akamai] Add Akamai package

### LINKED ISSUE #1304: Akamai
### Description
Akamai is the leading content delivery network (CDN) services provider for media and software delivery, and cloud security solutions. Their security offerings include Kona Site Defender, Client Reputation, Web Application, and Bot Manager.

### Architecture
Customers running any of the products listed above, events can be exported to a SIEM via the [Akamai SIEM API](https://developer.akamai.com/api/cloud_security/siem/v1.html). 

# Integration release checklist

This checklist is intended for integrations maintainers to ensure consistency
when creating or updating a Package, Module or Dataset for an Integration.

### All changes

- [ ] Change follows the [contributing guidelines](https://github.com/elastic/integrations/blob/master/CONTRIBUTING.md)
- [ ] Supported versions of the monitoring target are documented
- [ ] Supported operating systems are documented (if applicable)
- [ ] Integration or [System tests](https://github.com/elastic/elastic-package/blob/master/docs/howto/system_testing.md) exist
- [ ] Documentation exists
- [ ] Fields follow [ECS](https://github.com/elastic/ecs) and [naming conventions](https://www.elastic.co/guide/en/beats/devguide/master/event-conventions.html)
- [ ] At least a manual test with ES / Kibana / Agent has been performed.
- [ ] Required Kibana version set to:

### New Package

- [ ] Screenshot of the "Add Integration" page on Fleet added

### Dashboards changes

- [ ] Dashboards exists
- [ ] Screenshots added or updated
- [ ] Datastream filters added to visualizations

### Log dataset changes

- [ ] [Pipeline tests](https://github.com/elastic/elastic-package/blob/master/docs/howto/pipeline_testing.md) exist (if applicable)
- [ ] Generated output for at least 1 log file exists
- [ ] Sample event (`sample_event.json`) exists

### PR DESCRIPTION
**Pull request by legoguy1000**
## What does this PR do?

Add initial Akamai package to read SIEM logs

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's `changelog.yml` file.
- [x] If I'm introducing a new feature, I have modified the Kibana version constraint in my package's `manifest.yml` file to point to the latest Elastic stack release (e.g. `^7.13.0`).

## Author's Checklist

- [ ]

## How to test this PR locally

```
cd integrations/packages/akamai
elastic-package build && elastic-package stack down && elastic-package stack up -d -v && eval "$(elastic-package stack shellinit)" && elastic-package test  -v
```

## Related issues

- Closes #1304 
- Blocked by elastic/beats#27873

### CODE REVIEW

File: packages/akamai/_dev/build/build.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+dependencies:
+  ecs:
+    reference: git@1.11
```
Comment (legoguy1000) [inline packages/akamai/_dev/build/build.yml:3]: Need to update to 1.12

File: packages/akamai/data_stream/siem/elasticsearch/ingest_pipeline/default.yml [hunk 2]
```diff
@@ -0,0 +1,413 @@
+---
+description: Pipeline for parsing cloudflare logs
+processors:
```
Comment (legoguy1000) [inline packages/akamai/data_stream/siem/elasticsearch/ingest_pipeline/default.yml:3]: Need to add `ecs.version` field

File: packages/akamai/data_stream/siem/elasticsearch/ingest_pipeline/default.yml [hunk 3]
```diff
@@ -0,0 +1,425 @@
+---
+description: Pipeline for parsing cloudflare logs
```
Comment (efd6) [inline packages/akamai/data_stream/siem/elasticsearch/ingest_pipeline/default.yml:2]: s/cloudflare/akamai/

**Review by P1llus — APPROVED** [review]

**Pull request merged**